CMakeLists.txt

```diff
@@ -0,0 +1,4 @@
+cmake_minimum_required(VERSION 3.4)
+project(logger)
```

Comment:
I tried to build and install but lib is not installed

Reply:
It is building fine , to use this lib it is required to make sample i.e. a executable . Sample , install and find package would be taken in different PR.